src/modules/quoiFeur/quoiFeur.helpers.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  ChannelType,
+  type ChatInputCommandInteraction,
+  Client,
+  Guild,
+  type Message,
+  Role,
+} from 'discord.js';
+
+import { cache } from '../../core/cache';
+import { removeEmoji, removePunctuation } from '../../helpers/regex.helper';
+
+const ONE_MINUTE = 1 * 60 * 1000;
+const MUTED_BY_BOT = 'Muted by bot';
```

Comment:
In case any moderator watch the role list, it's best to be more explicit.

```suggestion
const MUTED_BY_BOT = 'Muted for coubeh';
```

Reply:
I think it's good to be generic here, it may be use later for another feature and it's understandable like this

Reply:
If any other feature use this role, the role would be remove when the bot restart, so it's not good. The role has to be for this feature only until we find a solution for this.

Reply:
remove role on restart it's a need for the role not the feature (to unmute buggy case)

Reply:
changed, not enough important to argue